### PR TITLE
Use datetime

### PR DESCRIPTION
**Pull request by aososam**
- use date time
- add travel time divider
- add pickup max delay 
- fix list object referencing
- add missing equipment type
- use integer division for travel time

### CODE REVIEW

File: python/darpinstances/instance.py [hunk 1]
```diff
@@ -190,6 +194,7 @@ class EquipmentType(Enum):
     WHEELCHAIR = 2
     ELECTRIC_WHEELCHAIR = 3
     MEDICAL_STROLLER = 4
+    SPECIAL_NEEDS_STROLLER = 5
```
Comment (F-I-D-O) [inline python/darpinstances/instance.py]: this should be instead of medical stroller

File: python/darpinstances/instance.py [hunk 2]
```diff
@@ -283,6 +296,7 @@ def read_instance(filepath: Path, travel_time_provider: MatrixTravelTimeProvider
         line_string = infile.readline()
         action_id = 0
         index = 0
+        travel_time_divider = instance_config.get('travel_time_divider', 1)
```
Comment (F-I-D-O) [inline python/darpinstances/instance.py:297]: it has to work even without this property, have you tested it?

**Review by F-I-D-O — CHANGES_REQUESTED** [review]

File: python/darpinstances/instance.py (hunk 2 above)
Comment (aososam) [reply python/darpinstances/instance.py:297]: Yes. The default value is 1 if it is not present in the config file.

File: python/darpinstances/instance.py [hunk 3]
```diff
@@ -198,16 +202,21 @@ def map_equipment_type(equipment_str: str) -> EquipmentType:
         "STANDARD_SEAT": EquipmentType.STANDARD_SEAT,
         "WHEELCHAIR": EquipmentType.WHEELCHAIR,
         "ELECTRIC_WHEELCHAIR": EquipmentType.ELECTRIC_WHEELCHAIR,
-        "MEDICAL_STROLLER": EquipmentType.MEDICAL_STROLLER
+        "MEDICAL_STROLLER": EquipmentType.MEDICAL_STROLLER,
```
Comment (F-I-D-O) [inline python/darpinstances/instance.py]: here also get rid of MEDICAL_STROLLER

**Review by F-I-D-O — CHANGES_REQUESTED** [review]

**Review by F-I-D-O — APPROVED** [review]

**Pull request merged**